app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
I do not think it would work, the table looks like this:

```html
<table>
  <thead>
    <tr>
      <th>filter by this column</th>
      <th>do not filter by this column</th>
    </tr>
  <thead>
  <tbody>
     <tr>
       <td><span class="label label-info result-filter-tag">value to filter</span></td>
       <td><span class="label label-danger">value I do not filter</span></td>
     </tr>
     <!-- more rows to follow -->
  <tbody>
</table>
```

`label-{tag}` determines the row, but `result-filter-tag` decides the column, so they both work together to filter on one cell in each table row.